packages/react-dom/src/test-utils/ReactTestUtilsAct.js

```diff
@@ -22,193 +22,124 @@ const [
   getFiberCurrentPropsFromNode,
   enqueueStateRestore,
   restoreStateIfNeeded,
-  /* eslint-enable no-unused-vars */
   flushPassiveEffects,
+  /* eslint-enable no-unused-vars */
```

Comment:
Is this suppression actually necessary?

Reply:
I'll check, doesn't seem like it

Reply:
Ah it's because it uses destructuring to get the nth item, but the previous items aren't referenced. I'll switch it to regular array indexing instead.